### ISSUE TITLE
Scoring metrics in classification methods

### ISSUE THREAD

**Issue by saragrau** [cschlaffner/PROTzilla2#211]
- Figure out how the score() method works in sklearn. And if its always the same metric used
- Is an extra parameter needed for cross validation and grid search
- think a way to only have 1 scoring parameter if possible

NOTES
a. each classification instance has a method score, which provides a default evaluation criterion: score(X, y, sample_weight=None)
b. With GridSearchCV, RandomizedSearchCV and cross_validate it is possible to set the own scoring criterion (and also multiple)
c. Score metrics can also be called as a function, similar to score(), e.g precision_score(y_true, y_pred)

  
## Acceptance Criteria
- [ ]
